app/assets/javascripts/app/views/aspect_membership_view.js

```diff
@@ -13,8 +13,7 @@ app.views.AspectMembership = app.views.AspectsDropdown.extend({
 
   events: {
     "click ul.aspect_membership.dropdown-menu > li.aspect_selector": "_clickHandler",
-    "keypress ul.aspect_membership.dropdown-menu > li.aspect_selector": "_clickHandler",
-    "click ul.aspect_membership.dropdown-menu > li.newItem": "showModal"
+    "keypress ul.aspect_membership.dropdown-menu > li.aspect_selector": "_clickHandler"
```

Comment:
About this one, I think some irrelevant classes can be removed. Actually, a good practice to me would be to use id instead of classes and classes only when there's a need to match multiple elements. All classes that are not meaningfull for selecting should be eliminated.